runHarness/AppInstance/AuctionKubernetesAppInstance.pm

```diff
@@ -155,7 +155,11 @@ override 'getEdgeAddrsRef' => sub {
 	  	 $logger->error("There are no IP addresses for the Kubernetes nodes");
 		 exit 1;
 	  }	
-      my $httpPort = $cluster->kubernetesGetNodePortForPortNumber("app=auction,type=webServer", 80, $self->namespace);
+	  # The http port will only be used if the ssl parameter is true
+      my $httpPort = 80;
+      if (!$self->getParamValue('ssl')) {
+      	$cluster->kubernetesGetNodePortForPortNumber("app=auction,type=webServer", 80, $self->namespace);
```

Comment:
The return value can be ignored and just 80 used?

Reply:
No.  This is a bug.